core/src/main/java/com/datastax/oss/driver/internal/core/metadata/token/CDCToken.java

```diff
@@ -0,0 +1,17 @@
+package com.datastax.oss.driver.internal.core.metadata.token;
+
+import net.jcip.annotations.Immutable;
```

Comment:
I'm not a great fan of nonstandard annotations like this. Could we please remove it?